app/src/main/java/com/stevesoltys/seedvault/restore/RestoreViewModel.kt

```diff
@@ -64,9 +64,12 @@ import org.calyxos.backup.storage.restore.RestoreService.Companion.EXTRA_TIMESTA
 import org.calyxos.backup.storage.restore.RestoreService.Companion.EXTRA_USER_ID
 import org.calyxos.backup.storage.ui.restore.SnapshotViewModel
 import java.util.LinkedList
+import kotlin.math.min
 
 private val TAG = RestoreViewModel::class.java.simpleName
 
+internal const val PACKAGES_PER_CHUNK = 100
```

Comment:
I guess the challenge here is finding the right number. I've seen the binder transaction limit vary greatly depending on what's going on otherwise with the system.

Reply:
We can try to pick something conservative here and just try to tweak it if needed later.

* Now: Lots of apps may fail
* With this: More likely to succeed
* In the future, if needed: decrease this further?

Reply:
It's not like we're going to get any tangible benefit from setting it very high, right? Maybe we could just set it to 25 or something?